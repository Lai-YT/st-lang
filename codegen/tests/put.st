% strings
put "Hello", ",", " world!"

% ints
put 1, 2, 3 ..

% bools
put true, false
