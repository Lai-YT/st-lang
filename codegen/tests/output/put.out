Hello, world!
123truefalse
